packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx

```diff
@@ -64,14 +63,18 @@ export const ScenePreviewCameraNodeEditor: EditorComponentType = (props) => {
     EditorControlFunctions.commitTransformSave([props.entity])
   }
 
+  const scene = new Scene()
```

Comment:
we should put the scene in module scope to avoid creating a new one each render

Reply:
done